packages/design-system/src/components/web-components/ds-tabs/ds-tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import { define } from '../preactement/define';
+import { Tabs, TabPanel } from '../../Tabs';
+import { TabsProps } from '../../Tabs/Tabs';
+import { findElementsOfType } from '../../utilities/findElementsOfType';
+import { createElement } from 'react';
+
+const attributes = ['selected-id', 'default-selected-id', 'tablist-class-name', 'children'];
+
+interface WrapperProps extends Omit<TabsProps, 'children'> {
+  selectedId?: string;
+  defaultSelectedId?: string;
+  tablistClassName?: string;
+  children: TabsProps['children'];
+}
+
+const Wrapper = ({ children = [], ...otherProps }: WrapperProps) => {
+  function parseChildren(node) {
+    const elements = findElementsOfType(['ds-tab-panel'], node);
+
+    return elements.map((element) => {
+      const { children, ...attrs } = element.props || {};
+      return createElement(TabPanel, { ...attrs }, children);
+    });
+  }
```

Comment:
This pattern is really coming into its own! Your code here is nice and concise 👌